srcpkgs/nudoku/template

```diff
@@ -1,17 +1,20 @@
 # Template file for 'nudoku'
 pkgname=nudoku
-version=1.0.0
+version=2.0.0
 revision=1
+archs="armv7l i686 x86_64"
```

Comment:
why?

Reply:
because I thought it will not build for musl. Now I figured than in musl making "./configure --disable-nls" works, but I don't know how to put that in the template.

Reply:
looking in other templates I figured it out, now passes all checks.